src/main/java/io/xpring/payid/PayIDClient.java

```diff
@@ -0,0 +1,120 @@
+package io.xpring.payid;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.reflect.TypeToken;
+import io.xpring.payid.generated.*;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.interledger.spsp.PaymentPointer;
+
+import io.xpring.payid.generated.api.DefaultApi;
+import io.xpring.payid.generated.model.PaymentInformation;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Implements interaction with a PayID service.
+ * Warning:  This class is experimental and should not be used in production applications.
+ */
+public class PayIDClient {
+    /**
+     * The network this PayID client resolves on.
+     */
+    private XRPLNetwork network;
+
+    /**
+     * Whether to enable SSL Verification.
+     */
+    private boolean enableSSLVerification;
+
+    /**
+     * Initialize a new PayIDClient.
+     *
+     * @param network The network that addresses will be resolved on.
+     */
+    public PayIDClient(XRPLNetwork network) {
+        this.network = network;
+        this.enableSSLVerification = true;
+    }
+
+    /**
+     * Set whether to enable or disable SSL verification.
+     *
+     * Exposed for testing purposes.
+     */
+    public void setEnableSSLVerification(boolean enableSSLVerification) {
```

Comment:
Consider annotating this method as `@VisibleForTesting`. It's an annotation that Google uses in their libraries that has become a fairly common convention to use.